python/py-urwid/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        urwid urwid 2.1.2 release-
+version             2.6.16
```

Comment:
version should be put in the `github.setup` line as it was before

Reply:
Done.  Thanks I didn't realise the github group sets version itself.